### PR TITLE
Break up large modules into smaller files.

### PR DESCRIPTION
**Pull request by whoahbot**
This PR is just a restructuring of our Rust code into submodules, it does not include any functional changes to the library.

### CODE REVIEW

**Review by Psykopear — APPROVED** [review]

File: src/execution/mod.rs [hunk 1]
```diff
@@ -49,26 +52,29 @@ use crate::recovery::{ProgressReader, StateCollector};
 use crate::recovery::{RecoveryConfig, StepId};
 use crate::recovery::{RecoveryStoreSummary, StateWriter};
 use crate::recovery::{StateBytes, StateKey};
-use crate::recovery::{StateReader, StateUpdate};
+use crate::recovery::StateReader;
```
Comment (blakestier) [inline src/execution/mod.rs:55]: I don't think this is related to your PR but i'm curious about the way we're importing things. Why bundle some imports in `{}` and then do it again? Is this a formatting bug?

File: src/execution/mod.rs [hunk 2]
```diff
@@ -49,26 +52,29 @@ use crate::recovery::{ProgressReader, StateCollector};
...
+use std::time::Duration;
 use timely::communication::Allocate;
-use timely::dataflow::operators::generic::builder_rc::OperatorBuilder;
 use timely::dataflow::{operators::*, Stream};
 use timely::dataflow::{ProbeHandle, Scope};
 use timely::progress::Timestamp;
 use timely::worker::Worker;
-use timely::Data;
+
+use self::periodic_epoch::{periodic_epoch_source, PeriodicEpochConfig};
```
Comment (blakestier) [inline src/execution/mod.rs:73]: 😳 what the heck.

i like that we modularize it. I hadn't seen this tho

File: src/inputs/manual_input.rs [hunk 3]
```diff
@@ -0,0 +1,129 @@
+use std::task::Poll;
+
+use crate::pyo3_extensions::{TdPyAny, TdPyCallable, TdPyCoroIterator};
+use crate::recovery::StateBytes;
+use crate::{py_unwrap, try_unwrap};
+use pyo3::exceptions::{PyTypeError, PyValueError};
+use pyo3::prelude::*;
+
+use super::{InputConfig, InputReader};
```
Comment (blakestier) [inline src/inputs/manual_input.rs:9]: like drinking a cool glass of water

File: src/operators/mod.rs [hunk 4]
```diff
@@ -10,17 +10,16 @@
...
-use crate::window::WindowLogic;
 use crate::try_unwrap;
 use crate::{log_func, unwrap_any};
 use log::debug;
 use pyo3::exceptions::PyTypeError;
 use pyo3::prelude::*;
-use std::task::Poll;
-use std::time::Duration;
+
+pub(crate) mod fold_window;
```
Comment (blakestier) [inline src/operators/mod.rs:19]: 🤔 would it make sense for us to further compartmentalize stateful vs not stateful operators? Or too much?

**Review by blakestier — APPROVED** [review]
🥇 Thanks for doing this. A few questions, nothing blocking.

**Pull request merged**